RestaurantsNearMe/Data/YelpRequest/FetchingActor.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FetchingActor.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2024-02-04.
+//
+
+import Foundation
+
+/// Used by Transport clients to manage if a request is being made
+actor FetchingActor {
```

Comment:
Still getting used to integrating these more, but the idea is this can be used to facilitate a single request is being made at a time